html/partials/patterns.html.haml

```diff
@@ -6,7 +6,7 @@
         :scope => "patterns",
         :title => "Patterns",
         :count => "#{patterns.length} #{Machinery.pluralize(patterns.length, "pattern")}"
-      - if @description.packages && @description.packages.package_system == "dpkg"
+      - if @description.patterns.patterns_system == "dpkg"
```

Comment:
What happens if the value is not there for older descriptions? Is this protected by the format version check?